api/inventory/v1/agents.proto

```diff
@@ -668,7 +664,47 @@ message AddPMMAgentResponse {
 
 // Add/Change NodeExporter
 
-message AddNodeExporterRequest {
+message AddAgentRequest {
+  oneof exporter {
+    AddPMMAgentParams pmm_agent = 1;
+    AddNodeExporterParams node_exporter = 2;
+    AddMySQLdExporterParams mysqld_exporter = 3;
+    AddMongoDBExporterParams mongodb_exporter = 4;
+    AddPostgresExporterParams postgres_exporter = 5;
+    AddProxySQLExporterParams proxysql_exporter = 6;
+    AddExternalExporterParams external_exporter = 7;
+    AddRDSExporterParams rds_exporter = 8;
+    AddAzureDatabaseExporterParams azure_database_exporter = 9;
+    AddQANMySQLPerfSchemaAgentParams qan_mysql_perfschema_agent = 10;
+    AddQANMySQLSlowlogAgentParams qan_mysql_slowlog_agent = 11;
+    AddQANMongoDBProfilerAgentParams qan_mongodb_profiler_agent = 12;
+    AddQANPostgreSQLPgStatementsAgentParams qan_postgresql_pgstatements_agent = 13;
+    AddQANPostgreSQLPgStatMonitorAgentParams qan_postgresql_pgstatmonitor_agent = 14;
```

Comment:
Probably we can name this types just `SomethingParams`, instead of `AddSomethingParams`.